django_structlog/processors.py

```diff
@@ -1,7 +1,7 @@
 import structlog
 
 
-def inject_context_dict(_, __, event_dict):
+def inject_context_dict(logger, method_name, event_dict):
     """Add the structlog context dict to log events generated by the stdlib logging library.
 
     >>> import structlog
```

Comment:
Since this is now just a direct passthrough to `structlog.contextvars.merge_contextvars` it should either be marked as deprecated or just removed. Thoughts @jrobichaud ?

Reply:
I think documenting correctly in the upgrade guide would be enough.

Reply:
(Removal)

Reply:
What does "documenting correctly" mean in this context? Remove it and document it in the upgrade guide?